File: Reverse String Char wise and Word Wise/revword.py
str = "Welcome To Python World"
str = ' ' + str #This is to insert a space in the starting

temp_str="" # Will store the single words before pushing it in final string
final_str="" # This will be our final string

for i in str[::-1]:
	if(i!=' '):
		temp_str+=(i)
	else:
		final_str+=(temp_str[::-1] + ' ')
		temp_str = ""

print(final_str)

File: Reverse String Char wise and Word Wise/revword.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
	string str = "Welcome To CPP World";
	str.insert(str.begin(),' ');
	string str_rev, temp_rev_string;

	for(int i = str.size()-1; i >=0;i--)
	{
		if(str[i]!= ' ')
		{
			temp_rev_string.push_back(str[i]);
		}

		else
		{
			reverse(temp_rev_string.begin(),temp_rev_string.end());
			str_rev.append(temp_rev_string+' ');
			temp_rev_string = "";
		}

	}
	cout << str_rev << endl;

}